src/machinetalk/application/commandbase.cpp

```diff
@@ -353,7 +353,7 @@ void CommandBase::sendEmcAxisUnhome(Container &tx)
 
 void CommandBase::sendShutdown(Container &tx)
 {
-    sendCommandMessage(MT_SHUTDOWN, tx);
+    sendCommandMessage(MT_LAUNCHER_SHUTDOWN, tx);
```

Comment:
`MT_LAUNCHER_SHUTDOWN` is for the ApplicationLauncher I think. Is there a problem with `MT_SHUTDOWN`?